pygmt/src/ternary.py

```diff
@@ -0,0 +1,66 @@
+"""
+ternary - Plot data on ternary diagrams.
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    C="cmap",
+    F="center",
+    G="fill",
+    JX="width",
+    R="region",
+    S="style",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", T="sequence", c="sequence_comma", p="sequence")
+def ternary(self, table, **kwargs):
+    r"""
+    Reads (*a*,\ *b*,\ *c*\ [,\ *z*]) records from *table* and plots symbols at
+    those locations on a ternary diagram. If a symbol is selected and no symbol
+    size given, then we will interpret the fourth column of the input data as
+    symbol size. Symbols whose *size* is <= 0 are skipped. If no symbols are
+    specified then the symbol code (see **style** below) must be present as
+    last column in the input.  If **style** is not specified then we instead
+    plot lines or polygons.
+
+    Full option list at :gmt-docs:`ternary.html`
+
+    {aliases}
+
+    Parameters
+    ----------
+    table : str or list or {table-like}
+        Pass in either a file name to an ASCII data table, a Python list, a 2D
+        {table-classes}.
+    {J}
+    width : str
+        *width*. Set the width of the figure by passing a number, followed by
+        a unit (**i** for inches, **c** for centimeters).
```

Comment:
Please update the docstring of `width` following the official documentation at https://docs.generic-mapping-tools.org/latest/ternary.html#j

Reply:
I added the second line, but I think the first line doesn't make sense in the context (what would setting the width have to do with a valid linear plot?). I wanted to leave in my original line explaining how the width is set, as that is not clear from the GMT docstring.